doc/src/manual/environment-variables.md

```diff
@@ -0,0 +1,138 @@
+# Environment Variables in Julia
+
+The environment variables that Julia uses generally start with `JULIA`. If [`Base.versioninfo()`](https://github.com/JuliaLang/julia/blob/master/base/interactiveutil.jl#L247) is called with `verbose` equal to `true`, then the output will list all environment variables for which `JULIA` appears in the name.
+
+## File locations
+
+### `JULIA_HOME`
+The absolute path of the directory containing the Julia executable, which sets the global variable [`Base.JULIA_HOME`](https://github.com/JuliaLang/julia/blob/master/base/initdefs.jl#L90).  If `$JULIA_HOME` is not set, then `Base.JULIA_HOME` defaults to its compile-time value.
+
+The executable itself is one of 
+```
+$JULIA_HOME/julia
+$JULIA_HOME/julia-debug
+```
+by default.
+
+The global variable [`Base.DATAROOTDIR`](https://github.com/JuliaLang/julia/blob/master/base/Makefile#L62)  determines a relative path from `Base.JULIA_HOME` to the data directory associated with Julia; then the directory.  Then the path 
+```
+$JULIA_HOME/$DATAROOTDIR/julia/base
+```
+determines the directory in which Julia initially searches for source files (via [`Base.find_source_file()`](https://github.com/JuliaLang/julia/blob/master/base/loading.jl#L124)).
+
+Likewise, the global variable [`Base.SYSCONFDIR`](https://github.com/JuliaLang/julia/blob/master/base/Makefile#L61) internal variable determines a relative path to the configuration file directory. Then Julia searches for a `juliarc.jl` file at
+
+```
+$JULIA_HOME/$SYSCONFDIR/julia/juliarc.jl
+$JULIA_HOME/../etc/julia/juliarc.jl
+```
+by default (via [`Base.load_juliarc()`](https://github.com/JuliaLang/julia/blob/master/base/client.jl#L316) ).
```

Comment:
There's an extra space here after the URL and before the outer closing parenthesis.